shimmy/dm_control_compatibility.py

```diff
@@ -57,36 +57,18 @@ def reset(
             self.np_random = RandomState(seed=seed)
 
         timestep = self._env.reset()
-        obs = dm_obs2gym_obs(timestep.observation)
-        info = {
-            "timestep.discount": timestep.discount,
-            "timestep.step_type": timestep.step_type,
-        }
+
+        obs, reward, terminated, truncated, info = expose_timestep(timestep)
```

Comment:
I dislike the name `expose_timestep`, change to `dm_timestep2gym_step`??

Reply:
Sure